--- gnuplot-4.0.0/src/variable.c.pom	2004-04-13 19:24:03.000000000 +0200
+++ gnuplot-4.0.0/src/variable.c	2006-12-22 09:20:06.000000000 +0100
@@ -257,8 +257,8 @@
     { "$`kpsexpand '$TEXMFMAIN'`/fonts/type1!" },
     { "$`kpsexpand '$TEXMFLOCAL'`/fonts/type1!" },
     /* Linux paths */
-    { "/usr/X11R6/lib/X11/fonts/Type1" },
-    { "/usr/X11R6/lib/X11/fonts/truetype" },
+    { "/usr/share/X11/fonts/Type1" },
+    { "/usr/share/X11/fonts/TTF" },
     /* HP-UX */
     { "/usr/lib/X11/fonts!"},
     /* Ghostscript */
