diff -up gnuplot-5.0.6/demo/circles.dem.nolena gnuplot-5.0.6/demo/circles.dem
--- gnuplot-5.0.6/demo/circles.dem.nolena	2017-06-26 12:44:54.497798902 -0400
+++ gnuplot-5.0.6/demo/circles.dem	2017-06-26 12:45:05.006547459 -0400
@@ -98,21 +98,6 @@ pause -1 "Hit return to continue"
 
 reset
 set size ratio -1
-set title "Lena's key points"
-unset xtics
-unset ytics
-unset key
-unset border
-set yrange [] reverse
-
-plot 'lena.rgb' binary array=(128,128) dx=4 dy=4 format='%uchar' with rgbimage, \
-     'lena-keypoints.bin' binary format='%double' with circles lc rgb "yellow"
-
-pause -1 "Hit return to continue"
-
-
-reset
-set size ratio -1
 set title "Delaunay triangulation of Hemisphere points, some empty circles in red"
 unset key
 
diff -up gnuplot-5.0.6/demo/Makefile.am.nolena gnuplot-5.0.6/demo/Makefile.am
--- gnuplot-5.0.6/demo/Makefile.am.nolena	2017-06-26 12:44:33.146309782 -0400
+++ gnuplot-5.0.6/demo/Makefile.am	2017-06-26 12:44:43.322066306 -0400
@@ -64,7 +64,7 @@ epslatex:
 	dvips epslatex
 
 ##m4-files-begin
-EXTRA_DIST = Makefile.am.in lena-keypoints.bin scatter2.bin sine.bin \
+EXTRA_DIST = Makefile.am.in scatter2.bin sine.bin \
 using.bin gnuplot.cfg world.cor 1.dat 2.dat 3.dat arrowstyle.dat \
 asciimat.dat battery.dat big_peak.dat candlesticks.dat cities.dat \
 clip14in.dat ctg-y2.dat delaunay-edges.dat ellipses.dat empty-circles.dat \
@@ -97,7 +97,7 @@ stringvar.dem surface1.dem surface2.dem
 tics.dem timedat.dem transparent.dem transparent_solids.dem using.dem \
 utf8.dem varcolor.dem vector.dem world2.dem world.dem demo.edf density.fnc \
 hexa.fnc line.fnc reflect.fnc gen-random.inc stat.inc GM1_sugar.pdb \
-bldg.png GM1_bonds.r3d blutux.rgb lena.rgb sound.par sound2.par start.par \
+bldg.png GM1_bonds.r3d blutux.rgb sound.par sound2.par start.par \
 gnuplot.rot gnu-valley random-points gpdemos.tcl html 
 ##m4-files-end
 
diff -up gnuplot-5.0.6/demo/Makefile.in.nolena gnuplot-5.0.6/demo/Makefile.in
--- gnuplot-5.0.6/demo/Makefile.in.nolena	2017-06-26 12:43:56.569184965 -0400
+++ gnuplot-5.0.6/demo/Makefile.in	2017-06-26 12:44:17.562682652 -0400
@@ -331,7 +331,7 @@ SUBDIRS = $(PLUGIN_DIR)
 DIST_SUBDIRS = plugin
 DEMO = all.dem
 GNUPLOT = gnuplot
-EXTRA_DIST = Makefile.am.in lena-keypoints.bin scatter2.bin sine.bin \
+EXTRA_DIST = Makefile.am.in scatter2.bin sine.bin \
 using.bin gnuplot.cfg world.cor 1.dat 2.dat 3.dat arrowstyle.dat \
 asciimat.dat battery.dat big_peak.dat candlesticks.dat cities.dat \
 clip14in.dat ctg-y2.dat delaunay-edges.dat ellipses.dat empty-circles.dat \
@@ -364,7 +364,7 @@ stringvar.dem surface1.dem surface2.dem
 tics.dem timedat.dem transparent.dem transparent_solids.dem using.dem \
 utf8.dem varcolor.dem vector.dem world2.dem world.dem demo.edf density.fnc \
 hexa.fnc line.fnc reflect.fnc gen-random.inc stat.inc GM1_sugar.pdb \
-bldg.png GM1_bonds.r3d blutux.rgb lena.rgb sound.par sound2.par start.par \
+bldg.png GM1_bonds.r3d blutux.rgb sound.par sound2.par start.par \
 gnuplot.rot gnu-valley random-points gpdemos.tcl html 
 
 @BUILD_PLUGIN_TRUE@PLUGIN_DIR = plugin
diff -up gnuplot-5.0.6/demo/rgbalpha.dem.nolena gnuplot-5.0.6/demo/rgbalpha.dem
--- gnuplot-5.0.6/demo/rgbalpha.dem.nolena	2017-06-26 12:46:06.232082511 -0400
+++ gnuplot-5.0.6/demo/rgbalpha.dem	2017-06-26 12:46:37.431336005 -0400
@@ -18,8 +18,8 @@ set yrange [ 0. : 128. ]
 # Alpha =  linear gradient on x
 
 plot 100.*(.4+sin(x/5.)/(x/5.)) lw 5 title 'solid line', \
-     'lena.rgb' binary array=(128,128) format="%uchar" flipy using 1:2:3:(2.*column(0)) \
-                with rgbalpha title "Lena with linear\nalpha gradient"
+     'blutux.rgb' binary array=(128,128) format="%uchar" flipy using 1:2:3:(2.*column(0)) \
+                with rgbalpha title "Tux with linear\nalpha gradient"
 
 pause -1 "Hit return to continue"
 
@@ -28,8 +28,8 @@ pause -1 "Hit return to continue"
 focus(x,y) = ((column(0)-x)**2 + (column(-1)-(127-y))**2) > 400 ? 0 : 255
 
 plot 100.*(.4+sin(x/5.)/(x/5.)) lw 5 title 'solid line', \
-     'lena.rgb' binary array=(128,128) format="%uchar" flipy using 1:2:3:(focus(70,50)) \
-                with rgbalpha title "Lena with circular mask"
+     'blutux.rgb' binary array=(128,128) format="%uchar" flipy using 1:2:3:(focus(70,50)) \
+                with rgbalpha title "Tux with circular mask"
 
 pause -1 "Hit return to continue"
 reset
